### PR TITLE
[Merged by Bors] - feat: port Algebra.ContinuedFractions.Computation.Translations

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by Parcly-Taxel** [comment]
I have fixed all the errors, please make sure you're happy with them.

File: Mathlib/Algebra/ContinuedFractions/Computation/Translations.lean [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+
+/-- Shows how the entries of the sequence of the computed continued fraction can be obtained by the
+integer parts of the stream of integer and fractional parts.
+-/
+theorem nth_of_eq_some_of_succ_nth_intFractPair_stream {ifp_succ_n : IntFractPair K}
+    (stream_succ_nth_eq : IntFractPair.stream v (n + 1) = some ifp_succ_n) :
+    (of v).s.get? n = some ⟨1, ifp_succ_n.b⟩ := by
+  unfold of IntFractPair.seq1
+  simp only [Stream'.Seq.map_tail, Stream'.Seq.get?_tail, Stream'.Seq.map_get?]
+  simp only [Stream'.Seq.get?, stream_succ_nth_eq, Option.map_some']
```
Comment (mo271) [inline Mathlib/Algebra/ContinuedFractions/Computation/Translations.lean:248]:
```suggestion
  simp [Stream'.Seq.map_tail, Stream'.Seq.get?_tail, Stream'.Seq.map_get?, stream_succ_nth_eq]
```

**Comment by ChrisHughes24** [comment]
> I'm not sure about changing the `get?` to `nth`

I agree, the names should be changed back to have `get?` in the name. `get?` is the mathlib4 name for `nth` I believe.

**Comment by mo271** [comment]
> > I'm not sure about changing the `get?` to `nth`
> 
> I agree, the names should be changed back to have `get?` in the name. `get?` is the mathlib4 name for `nth` I believe.

Done, I changed back to the original state, please check that this is how it should be, @Parcly-Taxel

**Review by Parcly-Taxel — APPROVED** [review]

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**